components/Typography/src/UIFontDescriptor+MaterialTypography.m

```diff
@@ -36,24 +36,32 @@ + (nonnull UIFontDescriptor *)mdc_preferredFontDescriptorForMaterialTextStyle:
   // Store the system font family name to ensure that we load the system font.
   // If we do not explicitly include this UIFontDescriptorFamilyAttribute in the
   // FontDescriptor the OS will default to Helvetica.
-  static NSString *systemFontFamilyName;
+  static NSString *smallSystemFontFamilyName;
+  // For font size 20+, iOS returns a different font family. Query it separately.
+  static NSString *largeSystemFontFamilyName;
   static dispatch_once_t onceToken;
   dispatch_once(&onceToken, ^{
-    UIFont *systemFont;
+    UIFont *smallSystemFont;
+    UIFont *largeSystemFont;
     if ([UIFont respondsToSelector:@selector(systemFontOfSize:weight:)]) {
-      systemFont = [UIFont systemFontOfSize:12 weight:UIFontWeightRegular];
+      smallSystemFont = [UIFont systemFontOfSize:12 weight:UIFontWeightRegular];
+      largeSystemFont = [UIFont systemFontOfSize:20 weight:UIFontWeightRegular];
     } else {
       // TODO: Remove this fallback once we are 8.2+
-      systemFont = [UIFont systemFontOfSize:12];
+      smallSystemFont = [UIFont systemFontOfSize:12];
+      largeSystemFont = [UIFont systemFontOfSize:20];
     }
-    systemFontFamilyName = [systemFont.familyName copy];
+    smallSystemFontFamilyName = [smallSystemFont.familyName copy];
+    largeSystemFontFamilyName = [largeSystemFont.familyName copy];
   });
 
   NSDictionary *traits = @{ UIFontWeightTrait : @(materialTraits.weight) };
+  NSString *fontFamily =
+      materialTraits.pointSize < 19.5 ? smallSystemFontFamilyName : largeSystemFontFamilyName;
```

Comment:
My assumption is probably wrong here. 

I thought of a good pivot point, and didn't think of a case that people might use a half-point font. _20_ is concerning me a bit, since it is a floating point value (_double_ to be exact). So there is no guarantee that `pointSize` is not represented as 19.99999997, or is there?